dnf/util.py

```diff
@@ -47,6 +47,22 @@
 """DNF Utilities."""
 
 
+def _classify_specs(namespace, values):
```

Comment:
Please can you rename it. It parse arguments. Please can you add a description? Thanks a lot.